go.mod

```diff
@@ -3,16 +3,23 @@ module github.com/practo/k8s-worker-pod-autoscaler
 go 1.12
 
 require (
-	github.com/aws/aws-sdk-go v1.29.15
+	9fans.net/go v0.0.2 // indirect
+	github.com/aws/aws-sdk-go v1.21.8
+	github.com/beanstalkd/go-beanstalk v0.0.0-20190515041346-390b03b3064a // indirect
+	github.com/kisielk/errcheck v1.2.0 // indirect
+	github.com/kr/beanstalk v0.0.0-20180818045031-cae1762e4858 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/mpdroog/beanstalkd v0.0.0-20151231090228-bf387addf002
+	github.com/rogpeppe/godef v1.1.1 // indirect
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.3
 	github.com/spf13/viper v1.4.0
-	k8s.io/api v0.0.0
-	k8s.io/apiextensions-apiserver v0.0.0
-	k8s.io/apimachinery v0.0.0
-	k8s.io/client-go v0.0.0
-	k8s.io/code-generator v0.0.0-20190612205613-18da4a14b22b
+	golang.org/x/tools v0.0.0-20200226224502-204d844ad48d // indirect
+	k8s.io/api v0.0.0-20190726022912-69e1bce1dad5
+	k8s.io/apiextensions-apiserver v0.0.0-20190726024412-102230e288fd
+	k8s.io/apimachinery v0.0.0-20190726022757-641a75999153
+	k8s.io/client-go v0.0.0-20190726023111-a9c895e7f2ac
+	k8s.io/code-generator v0.0.0-20190726022633-14ba7d03f06f
```

Comment:
@sushant-115 i don't think you back merged with #55

Reply:
These were automatically updated by running `make`

Reply:
Its reverting the upgrades done on #55

Reply:
Done